src/components/ui/Cookie.tsx

```diff
@@ -0,0 +1,90 @@
+import { AccentText, CookieText } from 'components/ui/Text'
+import { useSnapshot } from 'valtio'
+import Button from 'components/ui/Button'
+import NotificationsStore from 'stores/NotificationsStore'
+import classnames, {
+  backgroundColor,
+  borderColor,
+  borderRadius,
+  borderWidth,
+  boxShadow,
+  boxShadowColor,
+  display,
+  flexDirection,
+  gap,
+  inset,
+  margin,
+  maxWidth,
+  padding,
+  position,
+  space,
+  zIndex,
+} from 'classnames/tailwind'
+
+const basicCardStyles = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  position('fixed'),
+  inset(
+    'left-4',
+    'right-4',
```

Comment:
You probably don't need both left and right

Reply:
They are needed to make small gaps between the card and edges